### PR TITLE
Code Quality: Introduced Files.App.Win32 to avoid duplication

### PR DESCRIPTION
**Pull request by 0x5bfa**
## Summary

To avoid duplicating the same generated code by CsWin32.

## PR Checklist

- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   _None_

## Screenshots

_None_

### CODE REVIEW

File: src/Files.App.Win32/NativeMethods.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+	"$schema": "https://aka.ms/CsWin32.schema.json",
+	"public": true
+}
```
Comment (hez2010) [inline src/Files.App.Win32/NativeMethods.json:4]: Can you add `"allowMarshaling": false` as well? We want to make sure we don't use any runtime marshalling, especially in the server.

File: src/Files.App.Win32/NativeMethods.txt [hunk 2]
```diff
@@ -44,4 +38,33 @@ MoveFileFromApp
...
 GetKeyState
-CreateDirectoryFromApp
\ No newline at end of file
+CreateDirectoryFromApp
+RoRegisterActivationFactories
+RoRevokeActivationFactories
+WindowsCreateString
+WindowsDeleteString
+RoInitialize
+RoUninitialize
```
Comment (hez2010) [inline src/Files.App.Win32/NativeMethods.txt:47]: This is no longer needed after #15229

**Review by hez2010 — CHANGES_REQUESTED** [review]
Otherwise LGTM as long as the project builds.

File: src/Files.App.Win32/NativeMethods.json (hunk 1 above)
Comment (0x5bfa) [reply src/Files.App.Win32/NativeMethods.json:4]: I added it in the file in Server project.
Do we want to override the setting for every project?

File: src/Files.App.Win32/NativeMethods.txt (hunk 2 above)
Comment (0x5bfa) [reply src/Files.App.Win32/NativeMethods.txt:47]: Alr!

File: src/Files.App.Win32/NativeMethods.json (hunk 1 above)
Comment (hez2010) [reply src/Files.App.Win32/NativeMethods.json:5]: I don't think you need to add this configuration in other projects. The generated types will be public in Files.App.Win32, and can be consumed directly by other projects.

Comment (0x5bfa) [reply src/Files.App.Win32/NativeMethods.json:5]: Sure

Comment (0x5bfa) [reply src/Files.App.Win32/NativeMethods.json:5]: public: true
doesn't work...

**Comment by 0x5bfa** [comment]
I’m not sure

**Pull request closed without merging**